symbols/overlay29.yml

```diff
@@ -6561,6 +6685,24 @@ overlay29:
         Note that this array is only used for the cardinal directions. The elements at odd indexes are unused and unconditionally set to 0.
         
         This array is used by the dungeon generation algorithm when generating room imperfections. See GenerateRoomImperfections.
+    - name: GUMMI_LIKE_MESSAGE_IDS
```

Comment:
Assuming this means what I think it means, can we use the term `STRING_IDS` for all three of these symbols rather than `MESSAGE_IDS`?